From 7639b9bbac53271ae3aabd6c92999c348d554e0f Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.3-7639b9bbac5-20211025T211159Z-avarab@gmail.com>
In-Reply-To: <cover-0.3-00000000000-20211025T211159Z-avarab@gmail.com>
References: <cover-0.3-00000000000-20211025T211159Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 25 Oct 2021 15:45:47 +0200
Subject: [PATCH 1/3] leak tests: mark t5701-git-serve.sh as passing
 SANITIZE=leak
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The "t5701-git-serve.sh" test passes when run under a git compiled
with SANITIZE=leak, let's mark it as such to add it to the
"linux-leaks" CI job.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t5701-git-serve.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/t5701-git-serve.sh b/t/t5701-git-serve.sh
index aa1827d841d..1896f671cb3 100755
--- a/t/t5701-git-serve.sh
+++ b/t/t5701-git-serve.sh
@@ -5,6 +5,7 @@ test_description='test protocol v2 server commands'
 GIT_TEST_DEFAULT_INITIAL_BRANCH_NAME=main
 export GIT_TEST_DEFAULT_INITIAL_BRANCH_NAME
 
+TEST_PASSES_SANITIZE_LEAK=true
 . ./test-lib.sh
 
 test_expect_success 'test capability advertisement' '
-- 
2.33.1.1511.gd15d1b313a6

